### PR TITLE
Fix for vendors formatting

### PR DESCRIPTION
**Pull request by jifeon**
Also suggested vendor paths moved to abstract env from phantom env, since the repo includes the files themselves.

### CODE REVIEW

File: lib/AbstractEnvironment.js [hunk 1]
```diff
@@ -77,6 +77,29 @@ class AbstractEnvironment {
...
+    return this._injectFiles(filePaths);
+  }
+
+  /**
+   * @abstract
+   * @param {Array.<string>} filePaths
+   * @protected
+   */
+  _injectFiles(filePaths) {
+    throw new Error('You must redefine _injectFiles method in child environment');
```
Comment (maZahaca) [inline lib/AbstractEnvironment.js:101]: What's about env which does not support any file injection?
JsDom for example

**Review by maZahaca — APPROVED** [review]

File: lib/AbstractEnvironment.js (hunk 1 above)
Comment (jifeon) [reply lib/AbstractEnvironment.js:101]: These envs will not invoke this function. Easy, right? ;)

Comment (maZahaca) [reply lib/AbstractEnvironment.js:101]: You're killing me by your intelligence. You're right!

Comment (jifeon) [reply lib/AbstractEnvironment.js:101]: 🐗

**Pull request merged**